features/bevragen/overzicht.md

```diff
@@ -0,0 +1,80 @@
+# Personen bevragen
+
+De BRP bevragen API maakt het mogelijk om gegevens van actuele personen in de basisregistratie personen (BRP) te raadplegen. De personen worden opgezocht op basis van hun identificerende gegevens.
+
+In de BRP wordt een persoon uniek geïdentificeerd met behulp van hun burgerservicenummer. Is de burgerservicenummer van de te raadplegen persoon/personen bekend, dan moet de [Raadpleeg persoon met burgerservicenummer](#raadplegen-van-personen) operatie worden gebruikt om de betreffende persoon/personen te raadplegen.
+
+Is de burgerservicenummer van de te raadplegen persoon/personen niet bekend, dan kan deze worden opgezocht met behulp van de [Zoek persoon](#zoeken-van-personen) operaties.
+
+## Wet bescherming persoonsgegevens
+
+De BRP bevragen API is ontworpen conform de REST principes. Om ook aan de Wet bescherming persoonsgegevens te conformeren zijn er concessies gedaan met betrekking tot het toepassen van de REST principes. De belangrijkste concessie is dat de POST methode en niet de GET methode wordt gebruikt om personen te bevragen. Dit zorgt er voor dat er geen [persoonlijk identificeerbare informatie (PII)](https://piwikpro.nl/blog/pii-niet-pii-en-persoonsgegevens/) terecht komen in de url van een request en daardoor ook niet in server logs.
```

Comment:
Is het handig de concessies die we hebben gedaan aan de REST principes ergens te documenteren?

Reply:
zeker. Ga ik ook doen. Dit is een eerste aanzet, vooral om te kijken of dit is wat we willen